pnpm-lock.yaml

```diff
@@ -22672,15 +22830,15 @@ packages:
     resolution: {integrity: sha512-DNK4ruAqtyHaN8Zne7PkBTO+dD1Lr0YfTduMqlIyjvQIoztBkUxrvL+hKeLW8NXFKHOq/2upkxuoS9znQ9bW9A==}
     deprecated: This version of 'buffer' is out-of-date. You must update to v4.9.2 or newer
     dependencies:
-      base64-js: 1.5.1
+      base64-js: 1.3.0
```

Comment:
Can't quite make sense of why this went to a previous version. Dependencies are probably still satisfied but this shouldn't have happened with the change we made for tinylicious?